mozilla_repo_urls/errors.py

```diff
@@ -8,8 +8,13 @@ def __init__(self, url_string) -> None:
 
 
 class UnsupportedPlatformError(RepoUrlsBaseError):
+    @property
+    def supported_platforms(self):
+        return list(self._supported_platforms)
+
     def __init__(self, url_string, platform, supported_platforms) -> None:
+        self._supported_platforms = supported_platforms
         super().__init__(
-            f"Unsupported platform. Got: {platform}. "
-            f"Expected: {supported_platforms}. URL: {url_string}"
+            f"Unsupported version control host. Got: {platform}. "
```

Comment:
Good call on communicating on hosts rather than platform. I agree it makes things easier to understand 👍 

Nit: per [1], we're now passing a host rather than a platform object. I'd suggest we rename `platform` into `host`, then. 

```suggestion
            f"Unsupported version control host. Got: {host}. "
```

[1] https://github.com/mozilla-releng/mozilla-repo-urls/pull/6/files#diff-3b1582b5ab46ad40a414ed592a20a14c85e6ccad8a4bef900dec491840691db1R31